Khan/wonder-blocks
# Add Khanmigo theming for buttons with icons

## Summary:
We want to allow buttons to have a khanmigo-themed version of the icons.
The idea is that icons will be wrapped by a rounded container and have
different focus/hover states (probably worth defining a pressed state as well).

Still waiting on design confirmation that this looks right when there
are icons on both sides.

More info in this slack thread:
https://khanacademy.slack.com/archives/C8Z9DSKC7/p1697490308355979

Issue: https://khanacademy.atlassian.net/browse/WB-1631

## Test plan:
Manual testing on Storybook (/button--khanmigo-theme)
For both the regular theme and the Khanmigo theme:
- Make sure everything looks normal when idle and all the icons are visible
- Make sure that the focus ring looks correct when tabbing through the buttons

Khanmigo theme:
- Confirm that the background circle is visible for the secondary and
  tertiary buttons.
- Confirm that the hoveer/active styles are as expected for all buttons.
